Rakefile

```diff
@@ -1,46 +1,19 @@
-require 'rake'
-require 'rspec/core/rake_task'
-require 'puppet-lint'
-require 'puppet-lint/tasks/release_test'
 require 'bundler/gem_tasks'
+require 'rubocop/rake_task'
+require 'github_changelog_generator/task'
```

Comment:
I'd keep this optional and implement fallback. This allows you to skip the development group in CI.

Reply:
Do you have an example for this suggestion?

Reply:
What I typically do is this:

```ruby
begin
  require 'github_changelog_generator/task'
rescue LoadError
  # Gem not present
else
  GitHubChangelogGenerator::RakeTask.new(:changelog)
end
```

The benefit of putting the definition in the else part is that you very narrowly scope the exception catching.

Reply:
Appreciate this thanks.